launcher/minecraft/mod/tasks/BasicFolderLoadTask.h

```diff
@@ -52,6 +53,11 @@ class BasicFolderLoadTask : public Task {
         m_dir.refresh();
         for (auto entry : m_dir.entryInfoList()) {
             auto filePath = entry.absoluteFilePath();
+#if defined Q_OS_WIN32
+            if (APPLICATION->checkQSavePath(filePath)) {
+                continue;
+            }
+#endif
```

Comment:
Why only do this check on windows? it seems reasonable to be to not try to parse a fine that is in the process of being saved on any os.

Reply:
You are right.
I focused on windows because the crash happens only there.